src/library/Uno.Material/Styles/Controls/v2/MediaPlayerElement.xaml

```diff
@@ -144,6 +144,11 @@
 								   BasedOn="{StaticResource DefaultCommandBarStyle}">
 								<Setter Property="Background" Value="Transparent" />
 								<Setter Property="DefaultLabelPosition" Value="Collapsed" />
+								<!--
+									This ensures the command bar has the right height avoiding the issue seen on
+									https://github.com/unoplatform/uno-private/issues/1163
+								-->
+								<Setter Property="Height" Value="{ThemeResource NavBarAppBarThemeCompactHeight}" />
```

Comment:
This would mean we would need to have that resource present in the app and that's only the case if Material Toolkit is included in the project. And from this point, we can't assume that. Material itself can be included without Toolkit

Reply:
If we force this the `AppBarThemeCompactHeight` instead of the NavBar specific one, does it work in both cases of Material Toolkit being present or not?

Reply:
> If we force this the `AppBarThemeCompactHeight` instead of the NavBar specific one, does it work in both cases of Material Toolkit being present or not?

I will need to check. Let me validate and will come back to you

Reply:
> This would mean we would need to have that resource present in the app and that's only the case if Material Toolkit is included in the project. And from this point, we can't assume that. Material itself can be included without Toolkit

Right, understand

Reply:
Updated PR with a different fix